examples/experimental/persistent-volume-provisioning/README.md

```diff
@@ -0,0 +1,138 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/examples/experimental/persistent-volume-provisioning/README.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## Persistent Volume Provisioning
+
+This example shows how to use experimental persistent volume provisioning.
+
+### Pre-requisites
+
+This example assumes that you have an understanding of Kubernetes administration and can modify the scripts that launch kube-controller-manager.
+
+### Admin Configuration
+
+Admins configure their clusters with Provisioners tied to quality-of-service tiers.  The correct configuration will be
+the tier name and the volume plugin responsible for provisioning that request separated by a slash.  The configuration is
+added as a CLI flag to kube-controller-manager.
+
+For example, an admin running in AWS can add tier "foo" that provisioned using EBS:
+
+`--qos-tier="foo/kubernetes.io/aws-ebs"`
```

Comment:
I would like this flag to be renamed or prefaced to strongly correlate that it is storage related.